docs/release-notes/8.13.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[release-notes-header-8.13.0]]
+== 8.13
+
+[discrete]
+[[release-notes-8.13.0]]
+=== 8.13.0
+
+[discrete]
+[[features-8.13.0]]
+==== Features
+
+* Allows you to define an entity's (such as a host's or user's) `Asset criticality`, which can affect risk scores ({pull}176815[#176815], {pull}176294[#176294], {pull}172417[#172417], {pull}176056[#176056]).
+* Information on the Data Quality dashboard now persists in {elastic-sec} rather than disappearing after each session ({pull}175673[#175673], {pull}173185[#173185]).
+* Adds line-by-line diffs to the rules upgrade flyout so you can see what's changed between versions ({pull}174564[#174564]).
+* Adds alert suppression to the Indicator Match rule type ({pull}174241[#174241]).
+
+
+[discrete]
+[[enhancements-8.13.0]]
+==== Enhancements
+
+* Adds a date and dataview selector to Analyzer regardless of where it appears in {elastic-sec} ({pull}176364[#176364]).
+* Enables the newly expanded Host and User details flyouts by default ({pull}175899[#175899]).
+* Updates the Alert details flyout's header UI ({pull}175075[#175075]).
+* Improves the UI for information panels appearing in Analyzer ({pull}172397[#172397]).
+
+[discrete]
+[[bug-fixes-8.13.0]]
+==== Bug fixes
+
+* Fixes a bug that could prevent Analyzer from properly displaying particular alerts ({pull}178389[#178389]).
+* Fixes a bug that meant you couldn't edit, add, or remove filters to a custom rule ({pull}178207[#178207]).
+* Fixes a bug that caused error messages to appear on the MITRE ATT&CK Coverage Overview dashboard ({pull}178126[#178126]).
+* Prevents an infinite loading state on the Add Rules page for users with limited permissions ({pull}178005[#178005]).
+* Fixes a bug that prevented the **Reset Fields** action on the Alerts table from resetting the table's columns ({pull}177986[#177986]).
+* Fixes a bug that interfered with the rule filtering interface when you opened it from specific parts of {elastic-sec} ({pull}177946[#177946]).
+* Adds translated versions of risk score preview table columns ({pull}177680[#177680]).
+* Fixes a bug that could prevent the correct value of the `kibana.alert.threshold_result.terms.value` field from appearing in the Alert details flyout ({pull}177472[#177472]).
+* Fixes multiple bugs affecting the rule filters UI in the Create rule interface ({pull}177081[#177081]).
```

Comment:
@yara and @e40pud are 177081 and 178207 related? If they are, can we combine the summaries into a single one?

Reply:
@nastasha-solomon These are fixes of different bugs

Reply:
Gotcha - thanks, @banderror! I'll keep the summaries separate and resolve this thread.